editor/src/clj/editor/camera_editor.clj

```diff
@@ -337,8 +277,8 @@
     :fov (:fov camera)
     :near-z (:near-z camera)
     :far-z (:far-z camera)
-    :auto-aspect-ratio (not (zero? (:auto-aspect-ratio camera)))
-    :orthographic-projection (not (zero? (:orthographic-projection camera)))
+    :auto-aspect-ratio (not= 0 (:auto-aspect-ratio camera))
+    :orthographic-projection (not= 0 (:orthographic-projection camera))
```

Comment:
The use of `zero?` was causing a boxed math warning. This does not, since it can compare anything to anything. Thats's fine here, since this is only called at load-time.

Reply:
I don't think the check happens when the value is boxed, but when a boxed value is unboxed for a primitive operation. For `zero?`, I would imagine it does a primitive comparison internally.